tests/suites/1_stateful/09_http_handler/09_0006_route_error.result

```diff
@@ -0,0 +1,15 @@
+# error
+## page
+{"error":{"code":"404","message":"route error: query QID SHOULD be on server XXX, but current server is NODE, which started ... ago)"}}
+## kill
+{"error":{"code":"404","message":"route error: query QID SHOULD be on server XXX, but current server is NODE, which started ... ago)"}}
+## final
+{"error":{"code":"404","message":"route error: query QID SHOULD be on server XXX, but current server is NODE, which started ... ago)"}}
```

Comment:
@hantmac please take a look about whether the drivers could handle this error response into a proper exception with these message displayed.

Reply:
client cannot handle it, and it is hard to explain it in the error message, current message gives info helpful for debugging.

 we can later add a section about the routing problem in the doc, and in the error message let user to refer to the section.